src/main/RP.ts

```diff
@@ -119,8 +124,10 @@ function createVerifyResponseOptsFromBuilderOrExistingOpts(opts: { builder?: RPB
         verification: {
           mode: VerificationMode.INTERNAL,
           resolveOpts: {
-            didMethods: opts.builder.didMethods,
-            resolver: opts.builder.resolver ? getResolver({ resolver: opts.builder.resolver }) : getResolver({ didMethods: opts.builder.didMethods }),
+            didMethods: opts.builder.subjectSyntaxTypesSupported,
```

Comment:
Needs a did: filter in there, as subjectTypesSupported is bigger than DIDs alone. Also needs a TODO to handle other types like JWK-thumbprint

Reply:
added